SimpleServlet/src/main/java/com/skankhunt220/api/controller/UserController.java

```diff
@@ -0,0 +1,68 @@
+package main.java.com.skankhunt220.api.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import main.java.com.skankhunt220.entities.User;
+import main.java.com.skankhunt220.service.UserService;
+
+@WebServlet("/User") // servlet url
+public class UserController extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+	private static final UserService userService = new UserService();;
+	private User user;
+
+	private void sendResponse(String message, HttpServletRequest request, HttpServletResponse response)
```

Comment:
After removing the logic for getting `User` fields you don`t need a `request` parameter for this method